test/lib/BedrockTester.h

```diff
@@ -26,21 +57,32 @@ class BedrockTester {
     // Shuts down all bedrock servers associated with any testers.
     static void stopAll();
 
+    // This is a counter for available port numbers. This assumes that all ports are unused.
```

Comment:
a `counter` ? 😕

Reply:
Maybe counter is a bad description. I'll try and come up with better wording.